cli/src/trusted_operation.rs

```diff
@@ -104,7 +105,7 @@ fn execute_getter_from_cli_args<T: Decode + Debug>(
 	trusted_args: &TrustedCli,
 	getter: &Getter,
 ) -> TrustedOpResult<T> {
-	let shard = read_shard(trusted_args).unwrap();
+	let (_, shard) = get_identifiers(cli, trusted_args);
```

Comment:
this is out of context, sorry, but fixes an oversight of accepting the lack of the mrenclave arg